internal/database/repository/application.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/jackc/pgx/v5"
 )
 
-func (s *RepoPostgres) UpsertApplications(ctx context.Context, apps []*dao.ApplicationDAOInfo) error {
+func (s *PostgresRepository) UpsertApplications(ctx context.Context, apps []*dao.ApplicationDAOInfo) error {
```

Comment:
My opinion is that this is not an improvement. I know Kubernetes likes to spell out IncrediblyLongIdentifiersForEverythingBecauseStorageAndTypingPatienceIsInfinite but traditional Go code is usually quite succint, and they prefer reasonably short identifiers.